rclcpp/include/rclcpp/time.hpp

```diff
@@ -189,7 +189,7 @@ class Time
    */
   RCLCPP_PUBLIC
   static Time
-  max();
+  max(rcl_clock_type_t clock_type = RCL_SYSTEM_TIME);  // NOLINT
```

Comment:
I don't know the answer to this, so this is just a question: does adding a default parameter to a method change the ABI?  I would guess it does, but I'm not 100% sure about it.

Reply:
ah, i think we are introducing `default` argument here, that means changing `default` argument to the method, that is not guaranteed as ABI compatible change. (off topic, It would be nice if we have API/ABI checker with github workflow to add the lable on each PR for future enhancement.)

thanks for reminding me of that. i will go ahead to close this, and https://github.com/ros2/rclcpp/pull/2354